docker/dev/docker.env

```diff
@@ -1,6 +1,6 @@
 AWS_ACCESS_KEY_ID=x
 AWS_SECRET_ACCESS_KEY=x
-AWS_DEFAULT_REGION=us-east-1
+AWS_DEFAULT_REGION=us-east-2
```

Comment:
@RishabhJain2018 is this change going to affect any of the other running webserver or RDS connection if we change here?

@Ayukha to be safe, lets hard code the region in the function itself where you create the nodegroup.

Reply:
@Ayukha Can you please remove this change from this PR and hardcode it in the function itself.